readme.md

```diff
@@ -243,6 +243,15 @@ Determines if a `got.HTTPError` is thrown for error responses (non-2xx status co
 
 If this is disabled, requests that encounter an error status code will be resolved with the `response` instead of throwing. This may be useful if you are checking for resource availability and are expecting error responses.
 
+###### finalize
```

Comment:
I think `beforeRequest` would be a clearer name.

Reply:
I'm ok with that. I chose `finalize` to indicate that this is the very last thing that happens before the request is sent.

I'll rename.